### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by psychemedia**
Typos and small readability changes

### CODE REVIEW

File: ten-simple-rules-dockerfiles.Rmd [hunk 1]
```diff
@@ -365,11 +365,11 @@ To support both "one click execution" and to allow for custom configuration, it'
...
 # 8. Establish templates for new projects
 
-It's likely going to be the case that you develop interactive environments, servers, or workflows that are similar in nature.
+It's likely going to be the case that over time you will develop interactive environments, servers, or workflows that are similar in nature to each other.
 In this case, you should consider adopting a standard workflow that will give you a clean slate for a new project.
 As an example, cookie cutter templates (e.g., https://github.com/cookiecutter/cookiecutter) and project starter kits can provide files, directory organization, and build instructions that include basic steps for getting started.
 A good project template should get you started with a template for documentation, setting up testing via continuous integration, building a container, and even choosing a license.
-In the case of using a common `Dockerfile` or base, Docker's build caching will take shared lines into account and speed up build time.
+In the case of using a common `Dockerfile` or base, Docker's build cacheing will take shared lines into account and speed up build time.
```
Comment (vsoch) [inline ten-simple-rules-dockerfiles.Rmd]: This should be "caching" still I believe.

File: ten-simple-rules-dockerfiles.Rmd [hunk 2]
```diff
@@ -365,11 +365,11 @@ To support both "one click execution" and to allow for custom configuration, it'
 <!-- SUGGESTION for rule 8 instead of redundant comments with build commands - be about having a workflow to start a new project-->
 # 8. Establish templates for new projects
 
-It's likely going to be the case that you develop interactive environments, servers, or workflows that are similar in nature.
+It's likely going to be the case that over time you will develop interactive environments, servers, or workflows that are similar in nature to each other.
 In this case, you should consider adopting a standard workflow that will give you a clean slate for a new project.
```
Comment (vsoch) [inline ten-simple-rules-dockerfiles.Rmd]: to one another?

File: ten-simple-rules-dockerfiles.Rmd [hunk 3]
```diff
@@ -394,9 +394,9 @@ Consult Ten Simple Rules paper by Perez-Riverol et al. [@perez-riverol_ten_2016]
 # 10. Use the container daily, rebuild the image weekly {-}
 
 A container that is not regularly rebuilt risks not getting essential security updates, and serving as a risk for its users.
-If you are using a container built by a `Dockerfile` regularity in your work, it's recommended to set up continuous deployment, or a strategy to ensure that the container is regularly built and tested.
+If you are using a container built by a `Dockerfile` regularly in your work, it's recommended to set up continuous deployment, or a strategy to ensure that the container is regularly built and tested.
 While there is no hardened rule, it would be good practice to rebuild a container every once or two weeks that you use on a daily basis.
-In an idealistic world, containers that we built years ago should rebuild seamlessly, but this isn't the case, especially with rapidly changing technology relevant to machine learning and data science.
+In an idealistic world, containers that we built years ago should rebuild seamlessly, but this isn't neessarily the case, especially with rapidly changing technology relevant to machine learning and data science.
```
Comment (vsoch) [inline ten-simple-rules-dockerfiles.Rmd]: nessarily -> necessarily

Comment (nuest) [reply ten-simple-rules-dockerfiles.Rmd]: Fixed that one because it seemed obvious.

**Pull request merged**